docs/sdk/src/guides/enable_pov_reclaim.rs

```diff
@@ -0,0 +1,81 @@
+//! This guide will teach you how to enable storage weight reclaiming for a parachain. The
+//! explanations in this guide assume a project structure similar to the one detailed in
+//! the [substrate documentation](crate::polkadot_sdk::substrate#anatomy-of-a-binary-crate).
+//!
+//! # What is PoV reclaim?
+//! When a parachain submits a block to a relay chain like Polkadot or Kusama, it sends the block
+//! itself and a storage proof. Together they form the Proof-of-Validity (PoV). The PoV allows the
+//! relay chain to validate the parachain block by re-executing it. Relay chain
+//! validators distribute this PoV among themselves over the network. This distribution is costly
+//! and limits the size of the storage proof. The storage weight dimension of FRAME weights reflects
+//! this cost and limits the size of the storage proof. However, the storage weight determined
+//! during [benchmarking](`crate::reference_docs::frame_benchmarking_weight`) represents the worst
+//! case. In reality, runtime operations often consume less space in the storage proof. PoV reclaim
+//! offers a mechanism to reclaim the difference between the benchmarked worst-case and the real
+//! proof-size consumption.
+//!
+//!
+//! # How to enable PoV reclaim
+//! ## 1. Add the host function to your node
+//!
+//! To reclaim excess storage weight, a parachain runtime needs the
+//! ability to fetch the size of the storage proof from the node. The reclaim
+//! mechanism uses the
+//! [storage_proof_size](`cumulus_primitives_proof_size_hostfunction::storage_proof_size`)
```

Comment:
Hmm when I build locally, this link and other cumulus links below are broken, but I am unable to figure out why :thinking:. Anyone got an idea? They look correct to me.